man/systemd.exec.xml

```diff
@@ -3876,21 +3876,27 @@ StandardInputData=V2XigLJyZSBubyBzdHJhbmdlcnMgdG8gbG92ZQpZb3Uga25vdyB0aGUgcnVsZX
 
         <varlistentry>
           <term><varname>$MAINPID</varname></term>
+          <term><varname>$MAINPIDFDID</varname></term>
 
-          <listitem><para>The PID of the unit's main process if it is
-          known. This is only set for control processes as invoked by
+          <listitem><para>The UNIX process ID (PID) of the unit's main process if it is known, as well as the
+          64bit inode ID of the file descriptor returned by <citerefentry
+          project='man-pages'><refentrytitle>pidfd_open</refentrytitle><manvolnum>3</manvolnum></citerefentry>
+          for the main process (if supported). This is only set for control processes as invoked by
           <varname>ExecReload=</varname> and similar.</para>
 
-          <xi:include href="version-info.xml" xpointer="v209"/></listitem>
+          <xi:include href="version-info.xml" xpointer="v209"/>
+          <xi:include href="version-info.xml" xpointer="v258"/></listitem>
```

Comment:
Setting two version identifier looks spurious.
Maybe better to set custom version identifier, or split this into two entries.